### ISSUE TITLE
Empty page on view mode

### ISSUE THREAD

**Issue by Aveyron-RetD** [nextcloud/collectives#1328]
**Describe the bug**
I have one page who keeps getting empty on view mode.
When i click edit, the page goes back to being displayed

**To Reproduce**
Not sure if it is reproducible, but this page got put into a folder of the name of the page, and it's named readme and not put in the parent folder. This Readme page even has a tag pdf ?
Do not know how it got there....

![image](https://github.com/nextcloud/collectives/assets/53221471/271fc14f-655b-4efa-8c69-5840050d5fd7)

![image](https://github.com/nextcloud/collectives/assets/53221471/722087fd-5307-44ad-83cf-1e4f32ed1af3)

![image](https://github.com/nextcloud/collectives/assets/53221471/d1eeede8-0ba0-4b6a-8af9-8098b9d445de)

**Expected behavior**

**Screenshots**

**Server details:**
- Collectives app version: 2.11.0
- Nextcloud version: 29.0.1
- PHP Version: [e.g. 8.2]
- Database: [e.g. MariaDB 10.6]

**Client details:**
 - OS: [e.g. Windows/macOS/Ubuntu]
 - Browser: [e.g. Firefox, Chrome]
 - Browser version: [e.g. 22]
 - Device: [e.g. iPhone6, desktop]

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```

</details>

**Comment by Aveyron-RetD** [comment]
It's even weirder after putting back the file where it should be i still have an empty view.. Maybe the server needs time to sync ?

**Comment by mejo-** [comment]
Dear @Aveyron-RetD, I fear that I don't understand your exact problem.

Could you try to add some content to the file and then hit the save button (checkmark button at the top right) and see whether the content changes in view mode afterwards?

Maybe you can record a short screencast to better describe your problem?

**Comment by Aveyron-RetD** [comment]
Well it's not reproducible any more. I modified the file so much that it's normal. 
I could not reproduce it, so I tried multiple ways to trigger the bug, and now it's working. /-(

The behaviour was fairly consistent.
=> on edit mode : text was present with editor => Everything looked fine.
=> on preview mode: text was not present and I could not edit the file, the editor was not displayed. I could still edit the file if I was going to the folder

Will reopen if another file displays the same behaviour.
